doc/dev/crimson/pipelines.rst

```diff
@@ -0,0 +1,206 @@
+=========
+Pipelines
+=========
+
+Background
+==========
+
+Requests
+--------
+
+OSD serves different kinds of requests. Some of them are relatively easy to handle. For instance,
+the "ceph config get" command is directed straight to the config subsystem, and is fulfilled
+right away. But some are more "demanding". The most popular two of them are probably:
```

Comment:
I'm not sure demanding here is really relevant.  I think it makes more sense to center the document on:
1) rados protocol requirements
2) current crimson implementation
3) sensible future improvements